client/pool.c

```diff
@@ -470,6 +470,12 @@ static void close_connection(connection_list_element *connection, int index, con
 		ip & 0xff, ip >> 8 & 0xff, ip >> 16 & 0xff, ip >> 24 & 0xff, ntohs(port), message);
 }
 
+/* static void calculate_nopaid_shares infos:
++  struct connection_pool_data *conn_data is miner's side data
++  struct xdag_pool_task *task is pool's side data
++  xdag_hash_t hash is a digest computed by the pool starting from the
++  miner's data 'conn_data->data' and pool side context task->ctx0
+*/
```

Comment:
@sgaragagghu If wanna add descriptions for functions, please take care of format, not free style.
I think the doc HOW TO CONTRIBUTE should be available as soon as possible.

Should be something like this or other doc format.
/*
* @method calculate_nopaid_shares
* @param 
* struct connection_pool_data* connection data
* struct xdag_pool_task* task data
* xdag_hash_t a hash computed from conn_data->data and task-ctx0
* @return void
* @description calculate nopaid share per connection.
*/

Reply:
cool, will adapt it tomorrow

Reply:
tomorrow i hope to have time for it :)